x-pack/plugins/security_solution/public/common/components/header_global/index.tsx

```diff
@@ -91,6 +91,18 @@ export const HeaderGlobal = React.memo(
         },
         [navigateToApp, search]
       );
+
+      const hasCasesReadPermissions = useGetUserCasesPermissions()?.read;
+
+      // build a list of tabs to exclude
+      const tabsToExclude = new Set<string>([
```

Comment:
Remove cases from security solution's tabs if the user doesn't at least have read permissions.

@michaelolo24 @semd is this going to cause issues for the global nav work?

Reply:
We'll need to update to use this logic in our nav changes, but shouldn't be a huge change to merge these changes in to our side

Reply:
Also, I didn't see `hideDetectionEngine` actually being used, so I removed it in my PR